src/components/pages/course-form/CourseFormContainer.jsx

```diff
@@ -1,11 +1,102 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Form, Input, Space, Button } from 'antd';
+import { ModuleForm, ModuleCard } from '../module-form';
+import styled from 'styled-components';
+
+const StyledSpace = styled(Space)`
+  &&& {
+    width: 100%;
+  }
+`;
+
+// CourseFormContainer — No specific parent, props isn't coming down quite yet
+export default props => {
+  const [form] = Form.useForm();
+  const [modalVisible, setModalVisible] = useState(false);
+
+  const showModuleModal = () => setModalVisible(true);
```

Comment:
I love this Chaz. Nice work extrapolating out this functionality for the modal using very clear and understandable function names. This really helps keep the code maintainable for future devs!